### DISCUSSION TITLE
Playtesters Needed

### DISCUSSION THREAD

**Discussion by horganhoozen** [community/community#53052] [Other Feature Feedback, Questions, & Ideas]
Hello Everyone!

I just finished a playable version of my first game ever. I started building this game on a journey to learn about node.js, Javascript, HTML, and CSS. The code for this game was not AI-Generated. 

The game is called Capshuns and it is a simple game where 3-5 players submit captions for a random image. The players then vote on which caption they like the best. The first player to win 5 rounds, wins the game. I have tested it quite a bit during the development process but I can't find all the issues on my own. Please play a few games with your friends on it and let me know what improvements, changes, or issues you can think of. The game is web-based and should be playable on any device. If you see rendering issues on a specific device, let me know so I can address that. 

Thank you and enjoy!

The web address for the game:
https://capshuns-7q5dq.ondigitalocean.app/

**Comment by riatorz** [comment]
It redirect to main page when i write on "Type your message here" text area and press enter. 
Write a function about press enter or for text field thing..
I don't know tech stack so i can't suggest anything about your project. 
GL Let me know if you solve your problem.

**Reply by horganhoozen** [reply]
Thanks for the feedback :)

The server is running node.js and uses socket.io to communicate with the clients.
the client interface is just HTML, CSS, and Javascript.

I think the issue with the enter key has to do with form submission actions.

**Reply by horganhoozen** [reply]
I fixed the issue with the page reloading. It was that the default form action is to submit the form data instead of running the function that is supposed to send the message. I added an event listener for the Enter key while the focus is on that input box. I have tested it on a PC and it is working now. Thanks for the feedback and I hope you can try out the game for real this time LOL

**Reply by riatorz** [reply]
I tried recently, its working! Well done.

**Comment by riatorz** [comment]
When you create a room, the terminal says he has left the room. even if you're already in session. 
Maybe add a user at table and don't pop in that table just change in the right of the user row can be a small tag which like "in the game" or "in the room"..
![image](https://user-images.githubusercontent.com/44462050/232634201-c7e69136-5367-4042-83a2-53876e4ce6db.png)

**Reply by horganhoozen** [reply]
when you say terminal, do you mean the chat window?

**Reply by riatorz** [reply]
yes, chat window.

**Reply by horganhoozen** [reply]
I understand now. Instead of the verbiage " xxx has left the room." it would look better to say " xxx has joined a game."

**Reply by riatorz** [reply]
Right..
As i mentioned the image in the chat window, 3 prompt attract the attention:
"Dave has left the room"
"ria has left the room"
"Melissa has left the room"
Those prompt are about joining the created room by invite. Also prompt must be like this:
"Dave has joined the game", "Dave joined the room" etc..
In my opinion, if you do like this it'll be more clear for user experience..

**Reply by horganhoozen** [reply]
I have updated this to reflect the new verbiage :)
